interview-questions.md

```diff
@@ -9,59 +9,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer:`this` is a reference to key: value pair in an object.  It can be called upon using the .this
 
-  Researched answer:
+  Researched answer:this` is a reference to key: value pair in an object.  It can be called upon using the .this.  Can be used muiltple times 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a type of JS language that ties in JS and HTML. You should use it if you want a website that is dynamic and responded to a users input.
 
-  Researched answer:
+  Researched answer:  React is a JS libiary for putting together an user interface.  It is commonly used to build singe page and/or mobile applications
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: I belive that the lifecycle method that is required is render.
 
-  Researched answer:
+  Researched answer:  The render method is required in React.  It outputs HTML to the DOM.
 
 
 
 4. What is JSX? What is one notable difference between HTML and JSX?
 
-  Your answer:
+  Your answer:  One of the notable differences in JSX is the ability to use the combinatination of JS to make changes to the DOM.
 
-  Researched answer:
+  Researched answer: Althought there are some simulaties between HTML and JSX, in JSX you can write in JS methods and functions directly and boucne back between HTML and JS land to give you a more dynatic UI experance.
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: I belive that Yarn is a packaging software.  That gives you snipits of code and I think it changes the Json filies.
 
-  Researched answer:
+  Researched answer: Yarn is a package manger created by facebook. 
+  <!-- I couldn't find a decent answer -->
```

Comment:
Yes, yarn will connect dependencies. The yarn.lock, package.json, and node_nodules will be updated.